Pinta.Core/Managers/IChromeManager.cs

```diff
@@ -30,7 +30,61 @@
 
 namespace Pinta.Core;
 
-public sealed class ChromeManager
+public interface IChromeManager
```

Comment:
Renaming this file seems inconsistent with the other files in folder, I'd leave it as `ChromeManager.cs`